### PR TITLE
feat: add the "scroll" indicator on hovering the Intro section in the…

### PR DESCRIPTION
**Pull request by DiogoSoaress**
## What it solves

This change replaces the automatic scroll animation with a scroll indicator, similar to the indicator used in the Home > Intro section.

## How this PR fixes it

- When hovering the Intro section in the Core page, a scroll indicator is displayed in the bottom of the Intro section
- Reverts https://github.com/safe-global/safe-homepage/pull/226 after feedback

Screenshot 2023-08-14 at 19 21 47

### CODE REVIEW

File: src/components/Core/Intro/index.tsx [hunk 1]
```diff
@@ -8,7 +8,8 @@ const CoreIntro = ({ title, text, link }: BaseBlock) => {
       <Grid
         container
         className={css.container}
-        spacing={{ xs: 6, md: '30px', xl: '50px' }}
+        rowSpacing={{ xs: 6 }}
+        columnSpacing={{ md: '30px', xl: '50px' }}
```
Comment (DiogoSoaress) [inline src/components/Core/Intro/index.tsx:12]: The way I found to center the `.scroll` element was removing the padding left on the second `<Grid item >` by splitting the spacing attribute

File: src/components/Core/Intro/styles.module.css [hunk 2]
```diff
@@ -19,9 +28,28 @@
...
+  background-position: center;
+  width: 20px;
+  height: 20px;
+  margin: auto;
+}
+
 @media (min-width: 600px) {
   .container {
-    margin-top: 100px;
+    padding-top: 30px;
```
Comment (DiogoSoaress) [inline src/components/Core/Intro/styles.module.css:52]: Extends the "hovering" area and pulls the section a bit up to display the "scroll" element in some resolutions.

File: src/components/Core/Intro/index.tsx [hunk 3]
```diff
@@ -33,6 +34,9 @@ const CoreIntro = ({ title, text, link }: BaseBlock) => {
             </Button>
           )}
         </Grid>
+        <Typography variant="caption" className={css.scroll}>
+          New Whitepaper
```
Comment (katspaugh) [inline src/components/Core/Intro/index.tsx:38]: Screenshot 2023-08-15 at 08 43 31

Nothing happens when I click on this.

Also, why is "whitepaper" capitalized here, and on the "Read whitepaper" button? Is it a proper name? Are we writing in Ye Olde English?

File: src/components/Core/Intro/styles.module.css [hunk 4]
```diff
@@ -3,6 +3,15 @@
   margin-bottom: 75px;
 }
 
+.container .scroll {
+  opacity: 0;
+  transition: opacity 0.5s ease-in;
+}
+
+.container:hover .scroll {
+  opacity: 1;
```
Comment (katspaugh) [inline src/components/Core/Intro/styles.module.css:12]: Can you also make it float up and down slightly?

File: src/components/Core/Intro/index.tsx (hunk 3 above)
Comment (DiogoSoaress) [reply src/components/Core/Intro/index.tsx:38]: It's just and indicator to scroll down, similar to the one we have in the Home page intro section. The capitalization is also to be consistent with that style.

Comment (katspaugh) [reply src/components/Core/Intro/index.tsx:38]: Well my "user" feedback is that I want to click on it but it's not working.

Comment (katspaugh) [reply src/components/Core/Intro/index.tsx:38]: On the homepage, it says "Scroll", so I know what to do.

Here it just says "New whitepaper" which I interpret as a link. If you want it to be an instruction, it should tell me what to do.

Comment (DiogoSoaress) [reply src/components/Core/Intro/index.tsx]: I've added an "on click" handler to scroll to the desired section.

File: src/components/Core/Intro/styles.module.css (hunk 4 above)
Comment (DiogoSoaress) [reply src/components/Core/Intro/styles.module.css:20]: Added the animation. Good suggestion!

**Review by katspaugh — APPROVED** [review]
Looks good, thanks!

**Pull request merged**